### PR TITLE
Standalone schemas

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Standalone schemas support in AoT and JiT

### CODE REVIEW

File: packages/core/test/acceptance/ng_module_spec.ts [hunk 1]
```diff
@@ -212,6 +212,39 @@ describe('NgModule', () => {
          expect(spy.calls.mostRecent().args[0]).toMatch(/'custom-el' is not a known element/);
        });
 
+    it('should log an error about unknown element for a standalone component without CUSTOM_ELEMENTS_SCHEMA',
```
Comment (alxhub) [inline packages/core/test/acceptance/ng_module_spec.ts:215]: Should these tests be in `ng_module_spec`? I don't think they're testing any behavior around NgModules.

Comment (pkozlowski-opensource) [reply packages/core/test/acceptance/ng_module_spec.ts:215]: Yeh, was going back and forth on it :-) On one hand wanted to co-locate a new test with the existing ones. On the other hand I do agree that those tests have not much to do with NgModule (other than the fact that schemas are configured through a NgModule).

No strong opinion here, probably the best thing to do here would be to pull all those schemas checks into a separate test file (to have standalone and non-standalone ones close together).

Comment (pkozlowski-opensource) [reply packages/core/test/acceptance/ng_module_spec.ts:215]: @alxhub let's merge this PR as-is and I will create a new file with schemas-focused acceptance tests in a separate PR.

**Review by alxhub — APPROVED** [review]

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: public-api

**Review by atscott — APPROVED** [review]
reviewed-for: public-api

**Review by alxhub — APPROVED** [review]
Reviewed-for: public-api

**Comment by atscott** [comment]
[presubmit](http://cl/444701561)

**Comment by atscott** [comment]
This PR was merged into the repository by commit 3d4548450a1393becc7ea244d16cea4d99b05393.

**Pull request closed without merging**